cinch/roles/jenkins_master/templates/sysconfig_jenkins

```diff
@@ -79,7 +79,7 @@ JENKINS_HTTPS_PORT="{{ https_enabled | ternary(8443, '') }}"
 # Path to the keystore in JKS format (as created by the JDK 'keytool').
 # Default is disabled.
 #
-JENKINS_HTTPS_KEYSTORE="{{ jenkins_https_keystore }}"
+JENKINS_HTTPS_KEYSTORE=""
```

Comment:
This variable, and the one below it, which are no longer in use, should be removed from the defaults/main.yml file.